api/v1alpha3/vspheremachine_types.go

```diff
@@ -37,6 +37,10 @@ type VSphereMachineSpec struct {
 	// vsphere://12345678-1234-1234-1234-123456789abc
 	// +optional
 	ProviderID *string `json:"providerID,omitempty"`
+
+	// Tags is an optional set of tags to add to an instance.
+	// +optional
+	Tags []string `json:"tags,omitempty"`
```

Comment:
I'd put these in the `VirtualMachineCloneSpec`

Reply:
can we make the tag category explicit also ? (this would enable key:value use cases)

Reply:
@yastij Do you have an example of how to attach a tag from a specific category?